lib/class-wp-theme-json-gutenberg.php

```diff
@@ -713,9 +713,17 @@ protected static function sanitize( $input, $valid_block_names, $valid_element_n
 		$schema_styles_blocks   = array();
 		$schema_settings_blocks = array();
 		foreach ( $valid_block_names as $block ) {
-			$schema_settings_blocks[ $block ]           = static::VALID_SETTINGS;
-			$schema_styles_blocks[ $block ]             = $styles_non_top_level;
-			$schema_styles_blocks[ $block ]['elements'] = $schema_styles_elements;
+			// Blocks can have multiple style variations, so we need to build the schema for each one.
```

Comment:
Tiny nit: during previous backporting I've gotten feedback to avoid using "we" in comments. Would something like the following work?

```suggestion
			// Build the schema for each block style variation.
```

Reply:
Oh yes that sounds much better, thanks!